master/internal/config/config.go

```diff
@@ -225,6 +238,14 @@ func (c *Config) Resolve() error {
 		c.ResourceManager.AgentRM.Scheduler = DefaultSchedulerConfig()
 	}
 
+	if c.Webhooks.SigningKey == "" {
+		b := make([]byte, 6)
+		if _, err := rand.Read(b); err != nil {
+			return err
+		}
+		c.Webhooks.SigningKey = hex.EncodeToString(b)
+	}
```

Comment:
Sorry if this was unclear but both are true. The signing key is required, but if one is not provided then it will be generated, in which case users will need to view their cluster configuration to see the generated key.